src/extension/background-script/actions/lnurl/auth.ts

```diff
@@ -61,31 +85,28 @@ export async function authFunction({
       },
     });
 
-    keyMaterialForSignature = signResponse.data.signature;
-  }
+    const keyMaterialForSignature = signResponse.data.signature;
 
-  // make sure we got a signature
-  if (!keyMaterialForSignature) {
-    throw new Error("Invalid Signature");
-  }
+    // make sure we got a signature
+    if (!keyMaterialForSignature) {
+      throw new Error("Invalid Signature");
+    }
 
-  const hashingKey = sha256(keyMaterialForSignature).toString(Hex);
-  const url = new URL(lnurlDetails.url);
-  if (!url.host || !hashingKey) {
-    throw new Error("Invalid input");
-  }
+    const hashingKey = sha256(keyMaterialForSignature).toString(Hex);
 
-  let linkingKeyPriv;
-  const { settings } = state.getState();
-  if (settings.isUsingLegacyLnurlAuthKey) {
-    linkingKeyPriv = hmacSHA256(url.host, hashingKey).toString(Hex);
-  } else {
-    linkingKeyPriv = hmacSHA256(url.host, Hex.parse(hashingKey)).toString(Hex);
+    const { settings } = state.getState();
+    if (settings.isUsingLegacyLnurlAuthKey) {
```

Comment:
Is there a reason to support this still with the new keys? cc @bumi

Reply:
@reneaaron  This code is only executed in the non-mnemonic LNURL auth